api/src/GTAVersions.Data/Entities/RefreshToken.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace GTAVersions.Data.Entities
+{
+    public class RefreshToken
```

Comment:
Why this model is in `GTAVersions.Data` assembly. In `GTAVersions.Data` store only Database related models